### ISSUE TITLE
Normalization to [-1, 1] for DeepLab

### ISSUE THREAD

**Issue by netw0rkf10w** [nyoki-mtl/pytorch-segmentation#23]
Hi @nyoki-mtl,

I would like to ask a quick question please. 

I see in the [code](https://github.com/nyoki-mtl/pytorch-segmentation/blob/ddf7ce97114afa0f18902ba4a8f6dd6db581bcec/src/dataset/cityscapes.py#L87) that, for DeepLab, you normalize the input image to the range `[-1, 1]` like this:

`image = (image/255)*2 - 1`

Could you please tell me how you came up with this? I did some experiments and this normalization indeed yields the best performance. I checked the official TensorFlow implementation but I couldn't find where they used this normalization.

Thank you in advance for your response!
